rules/rule.go

```diff
@@ -0,0 +1,58 @@
+//Package rules ...
+// Copyright (c) 2017 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package rules
+
+import (
+	"errors"
+	"github.com/intel-go/nff-go/common"
+)
+
+var (
+	pccFilters []PCCFilter
+	adcFilters []ADCFilter
+	sdfFilters []SDFFilter
+)
+
+var (
+	pccSDFMap map[int][]PCCFilter
+	pccADCMap map[int][]PCCFilter
+)
+
+//Init ... initial rules and prepare and install acl
+func Init() {
```

Comment:
This should be somewhere in example, not as part of public library file? (Am I wrong?)

Reply:
ok

Reply:
will need to review it

Reply:
taken care